src/PointwiseFunctions/AnalyticData/CurvedWaveEquation/ScalarWaveKerrSchild.hpp

```diff
@@ -0,0 +1,225 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <cstddef>
+#include <limits>
+#include <memory>
+#include <type_traits>
+#include <utility>
+
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Evolution/Systems/CurvedScalarWave/Tags.hpp"
+#include "Evolution/Systems/ScalarWave/Tags.hpp"
+#include "Options/Options.hpp"
+#include "PointwiseFunctions/AnalyticData/AnalyticData.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/GeneralRelativity/KerrSchild.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/WaveEquation/PlaneWave.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/WaveEquation/RegularSphericalWave.hpp"
+#include "PointwiseFunctions/GeneralRelativity/TagsDeclarations.hpp"
+#include "PointwiseFunctions/MathFunctions/MathFunction.hpp"
+#include "Utilities/Requires.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+#include "Utilities/TypeTraits.hpp"
+
+/// \cond
+namespace PUP {
+class er;  // IWYU pragma: keep
+}  // namespace PUP
+namespace Tags {
+template <typename Tag>
+struct dt;
+}  // namespace Tags
+/// \endcond
+
+// IWYU pragma: no_include <pup.h>
+
+namespace CurvedScalarWave {
+namespace AnalyticData {
+/*!
+ * \brief Analytic initial data for a 3D spherical wave in Kerr spacetime
+ *
+ * \details This constructs the evolved variables of the CurvedScalarWave
+ * system for a regular spherical wave in \f$ 3\f$ spatial dimensions with
+ * a Kerr black hole providing background spacetime curvature. The scalar
+ * field profile is obtained from any class in `ScalarWave::Solutions`,
+ * and \f$ 3+1\f$ spacetime variables are obtained from
+ * `gr::Solutions::KerrSchild`. These are used to construct \f$\{\psi, \Phi_i,
+ * \Pi\}\f$ using their definitions:
+ *
+ * \f{align}
+ * \psi &:= \overset{\circ}{\psi} \\
+ * \Phi_i &:= \partial_i\psi = \overset{\circ}{\partial_i\psi}
+ *                           = \overset{\circ}{\Phi_i} \\
+ * \Pi &:= -t^a \partial_a \psi = \frac{1}{\alpha}
+ *      \left(\beta^k \partial_k\psi - \partial_t\psi\right) \nonumber \\
+ *     &=  \frac{1}{\alpha}\left(\beta^k \Phi_k -
+ *                                \overset{\circ}{\partial_t\psi}\right)
+ * \f}
+ *
+ * where
+ *
+ * - the hollow dots \f$\overset{\circ}{}\f$ over quantities indicate that they
+ * have been obtained from `ScalarWave::Solutions` member class (in
+ * flat spacetime and using coordinate derivatives), and
+ * - \f$t^a\f$ is the unit normal to spatial slices that spacetime has been
+ * foliated into.
+ *
+ * That the coordinate profile of scalar field \f$\psi\f$ in curved spacetime is
```

Comment:
I don't think this level of detail is needed in an `analytic data` since this is just the definitions from the curved scalar wave system itself. If you want this explained somewhere, I feel like it should be on the CSW system itself, rather than an analytic data.

Reply:
Simplified the description